### PR TITLE
update 2022-06-03

### PR DESCRIPTION
**Pull request by kba**

### CODE REVIEW

**Review by stweil — CHANGES_REQUESTED** [review]
Fine for me, but I missed the changelog for ocrd_all itself.

**Comment by kba** [comment]
Good point, I'll amend the CHANGELOG.

**Review by bertsky — APPROVED** [review]
LGTM

**Comment by kba** [comment]
https://github.com/OCR-D/ocrd_all/pull/326/commits/1cb4a6057a2aead5ed9da52b927cd59db4f24281 added CHANGELOG for ocrd_all itself.

**Review by stweil — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,108 @@
 # Changelog
 
+## [v2022-06-02](https://github.com/OCR-D/ocrd_all/releases/v2022-06-02)
```
Comment (stweil) [inline CHANGELOG.md:3]: @kba, maybe update the date here.

Comment (kba) [reply CHANGELOG.md]: https://github.com/OCR-D/ocrd_all/pull/326/commits/4f070d4140702e3fbf6e4720b0864d63e94aa5f4 done

**Pull request merged**